sql/rpl_gtid.h

```diff
@@ -1344,6 +1345,14 @@ class Gtid_set {
     existing allocated memory will be re-used.
   */
   void clear();
+
+  /**
+    Removes the sidnos (and their intervals) from the logged state.
+
+    @params sidnos List of rpl_sidno to remove.
+  */
+  void remove(const std::vector<rpl_sidno> &sidnos);
```

Comment:
typedef/using for a custom container type.
```suggestion
  using rpl_sidno_container = std::vector<rpl_sidno>;
  void remove(const rpl_sidno_container &sidnos);
```